### PR TITLE
2016Q4.feature.docker-compose.groupadd

### PR DESCRIPTION
**Pull request by HangYin2015**
@chrberger Could you approve this PR? Now all use cases in opendlv.core related to the documentation camera have been updated to use group_add. This requires docker-compose 1.9.0.

### CODE REVIEW

File: usecases/latest/recording.can+documentationcamera/docker-compose.yml [hunk 1]
```diff
@@ -66,7 +66,7 @@ services:
             - proxy-camera
         volumes:
             - ~/recordings:/opt/recordings
-        user: root  # This parameter will be replaced as soon as group_add parameter is available in docker-compose, cf. https://github.com/docker/compose/pull/3709.
+        user: odv  # This parameter will be replaced as soon as group_add parameter is available in docker-compose, cf. https://github.com/docker/compose/pull/3709.
```
Comment (chrberger) [inline usecases/latest/recording.can+documentationcamera/docker-compose.yml]: This comment next to odv is misleading.

File: usecases/latest/recordings.documentationcamera+velodyne/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    # Micro-service for proxy-camera.
+    proxy-camera:
+        build: .
+        group_add:
+            - video
+        depends_on:
+            - odsupercomponent
+        devices:
+            - "/dev/video0:/dev/video0"
+        user: odv  # This parameter will be replaced as soon as group_add parameter is available in docker-compose, cf. https://github.com/docker/compose/pull/3709.
```
Comment (chrberger) [inline usecases/latest/recordings.documentationcamera+velodyne/docker-compose.yml]: The comment next to odv is misleading.

**Review by chrberger — CHANGES_REQUESTED** [review]

**Comment by HangYin2015** [comment]
Thanks for the comments. I should have removed the comments that I overlooked. Now they are removed.

**Comment by chrberger** [comment]
I have just looked at the PR but I still see some misleading comments? Could you please re-check?

**Review by chrberger — APPROVED** [review]

**Comment by HangYin2015** [comment]
Sorry that I still overlooked some of those comments. I did a thorough search and removed the remaining comments. There are still several places with the misleading comment on group_add in the use cases involving axis camera, which I did not test before.

**Pull request merged**